moped-editor/src/views/projects/projectView/ProjectComponentsMapEdit.js

```diff
@@ -414,7 +413,6 @@ const ProjectComponentsMapEdit = ({
       */
       name: "",
       project_id: Number.parseInt(projectId),
-      status_id: 1,
```

Comment:
Same here - there is no longer a need initialize `status_id` for components on creation.